src/Microsoft.AspNet.Mvc.ModelBinding/ModelStateDictionaryUtility.cs

```diff
@@ -0,0 +1,73 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.Framework.Internal;
+using System;
+using System.Linq;
+using System.Reflection;
+
+namespace Microsoft.AspNet.Mvc.ModelBinding
+{
+    /// <summary>
+    /// Utility class for clearing entries in <see cref="ModelStateDictionary"/>.
+    /// </summary>
+    public static class ModelStateDictionaryUtility
```

Comment:
Any time you find yourself reaching for a word like `Utility` or `Helper` it's usually a clue that there's a better option.